plugins/fuse/server/server_fuse.py

```diff
@@ -387,17 +387,21 @@ def unmountServerFuse(name):
     mount.
 
     :param name: a key within the list of known mounts.
+    :return: None for success or an error number from the unmount process
```

Comment:
Since a failure to unmount is an error condition, would it be better to raise an exception in that case? The calling function could always choose to do nothing, but it would force the caller to at least acknowledge that an error is occurring (as opposed to being able to just never examining the return value). Additionally, it removes a heterogeneously-valued return type.

This is really just a microexample of C-style error handling vs C++-style, and I think we should prefer the latter.